src/object.ts

```diff
@@ -0,0 +1,27 @@
+import { clonedMathFloor, clonedMathRandom, generatePercentChanceToFlake, log, PERCENT_CHANCE_OF_SUCCESS } from './utils';
+
+const clonedKeys = Object.keys;
+
+Object.keys = function (obj: object): string[] {
+    let keys = clonedKeys.call(this, obj);
+
+    if (clonedMathRandom() < 0.5) {
```

Comment:
let's shuffle based on generatePercentChanceToFlake.
This is so when someone makes `PERCENT_CHANCE_OF_SUCCESS` 100, they can completely disable all possible weirdness